sdk/eventhub/azure-eventhub/azure/eventhub/_configuration.py

```diff
@@ -29,7 +29,7 @@ def __init__(
         retry_backoff_max: int = 120,
         network_tracing: bool = False,
         http_proxy: Optional[Dict[str, Any]] = None,
-        transport_type: TransportType = TransportType.Amqp,
+        transport_type: TransportType = None,
```

Comment:
is this public API and could be breaking?

Reply:
it shouldn't be public